python/cudf/cudf/core/frame.py

```diff
@@ -30,6 +31,162 @@ def _gather(self, gather_map):
         result._copy_categories(self)
         return result
 
+    def _as_column(self):
+        """
+        _as_column : Converts a single columned Frame to Column
+        """
+        assert (
+            len(self._data) == 1
```

Comment:
use accessor instead:
```suggestion
            self. _num_columns == 1
```